mkdocs.yml

```diff
@@ -0,0 +1,43 @@
+site_name: HyP3 SDK
+site_url: https://ASFHyP3.github.io/hyp3-sdk/
+site_author: ASF APD/Tools Team
+site_description: A python wrapper around ASF's HyP3 API
+copyright: © 2020 Alaska Satellite Facility
+google_analytics: ['UA-991100-5', 'search.asf.alaska.edu']
```

Comment:
could drop GA for now, since at a quick glance it wasn't functional, and circle back to it later